src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -1116,25 +968,6 @@ def test_get_job_logs__job_id__no_job(self):
             self.jc._handle_comm_message(req_dict)
         self.check_error_message(req_dict, err)
 
-    @mock.patch(CLIENTS, get_mock_client)
-    def test_get_job_logs__job_id__bad_job(self):
```

Comment:
this does the same thing as the test below -- checks a job ID that returns a not found error => deleted